turkish_lm_tuner/t5_classifier.py

```diff
@@ -18,7 +21,12 @@ class T5ForClassification(T5PreTrainedModel):   # nn.Module
     def __init__(self, pretrained_model_name, config, num_labels, problem_type, dropout_prob=0.1):
         super().__init__(config)
 
-        self.encoder = T5EncoderModel.from_pretrained(pretrained_model_name)
+        try:
+            self.encoder = T5EncoderModel.from_pretrained(pretrained_model_name)
+        except Exception as e:
+            pretrained_model_name = config._name_or_path
+            self.encoder = T5EncoderModel.from_pretrained(pretrained_model_name)
+

```

Comment:
Do we really need exception handling here? When does the encoder initialization fail with `pretrained_model_name`?

Reply:
This is really bad code I realize :D but there was a problem with the `from_pretrained` method, when it calls the init of T5forSequenceClassification, that argument is overwritten as the config.. I couldn't solve it, so I added this.. If I can find a solution, I will fix it! 👍

Reply:
Can we predict if it's going to fail beforehand and update the pretrained_model_name accordingly? In what scenarios do we use the first from_pretrained method compared to the other one?